android/libraries/rib-android/gradle.properties

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2017. Uber Technologies
```

Comment:
Should we use copyright of Uber here?

Reply:
not sure looks like we should not remove it
![image](https://user-images.githubusercontent.com/725526/54757567-5dce3d00-4be2-11e9-98c9-9385e391bc58.png)

Reply:
I would double check, but let's leave it for now :)